internal/util/validateendpoints.go

```diff
@@ -24,90 +24,114 @@ import (
 	"strings"
 )
 
-var identityToken string
+// test holds the openapi3.Operation and HTTP method associated with a single
+type test struct {
+	operation  *openapi3.Operation
+	httpMethod string
+}
 
-// validateEndpoints tests all paths (represented by openapi3.Paths) with all HTTP methods and given response bodies
+// ValidateEndpoints tests all paths (represented by openapi3.Paths) with all HTTP methods and given response bodies
 // and make sure they respond with the expected status code. Returns a success bool based on whether all the tests
```

Comment:
Returning a bool here seems a little odd. Is a simple bool enough information for the caller to diagnose what happened with their test?